test/test.js

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+'use strict';
```

Comment:
I think this is the de-facto standard for non-module scripts if I get this right. Should the test check the API with the same semantic as users will use it in scripts?